packages/server/lib/modes/interactive.ts

```diff
@@ -158,6 +158,10 @@ export = {
   },
 
   async run (options: LaunchArgs, _loading: Promise<void>) {
+    // Note: We do not await the `_loading` promise here since initializing
+    // the data context can significantly delay initial render of the UI
+    // https://github.com/cypress-io/cypress/issues/26388#issuecomment-1492616609
```

Comment:
This is good context, but might be worth pointed to code vs a github comment in the event this is code ever changes so this stays "true" and/or relevant.